templates/zonefile.j2

```diff
@@ -9,51 +9,51 @@ $TTL 1W
 	IN	MX 10	smtp.{{ dns.clusterid }}.{{ dns.domain | lower }}.
 ;
 ; 
-ns1	IN	A	{{ helper.ipaddr }}
-smtp	IN	A	{{ helper.ipaddr }}
+ns1			IN	A	{{ helper.ipaddr }}
+smtp			IN	A	{{ helper.ipaddr }}
 ;
-{{ helper.name | lower }}	IN	A	{{ helper.ipaddr }}
+{{ helper.name | lower }}			IN	A	{{ helper.ipaddr }}
 ;
 ; The api points to the IP of your load balancer
-api		IN	A	{{ helper.ipaddr }}
-api-int		IN	A	{{ helper.ipaddr }}
+api			IN	A	{{ helper.ipaddr }}
+api-int			IN	A	{{ helper.ipaddr }}
 ;
 ; The wildcard also points to the load balancer
-*.apps		IN	A	{{ helper.ipaddr }}
+*.apps			IN	CNAME	{{ helper.name | lower }}.{{ dns.clusterid }}.{{ dns.domain | lower }}.
 ;
 {%if setup_registry %}
 ; Create entry for the local registry
-{{ registry_host }}	IN	A	{{ helper.ipaddr }}
+{{ registry_host.split('.')[0] }}		IN	A	{{ helper.ipaddr }}
```

Comment:
Why is the split needed?

Reply:
Discovered in my install that registry_host is a FQDN.

Reply:
The registry_host fqdn is set in tasks/set_facts.yaml

Reply:
@christianh814 I dont think this change is needed
as we already have
`registry	IN	A	{{ helper.ipaddr }}` on master branch code (maybe the devel branch is on previous code state which we changed later)